### PR TITLE
Bumped the comments version again

### PR DESCRIPTION
**Pull request by richard-still-ft**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "o-gallery": "^1.6.1",
     "o-date": "^1.1.1",
     "n-video": "^2.5.1",
-    "o-comments": "2.0.5",
+    "o-comments": "2.0.6",
```
Comment (ironsidevsquincy) [inline bower.json]: any reason this is fixed?

Comment (richard-still-ft) [inline bower.json]: You mean the version? Not that I know of... I assumed there was so left it that way but it would nice to allow patch versions.

Comment (matthew-andrews) [inline bower.json]: yes there is

Comment (matthew-andrews) [inline bower.json]: *needs* to be kept in sync with this here:-
https://github.com/Financial-Times/next-article/blob/132588b6f756cd701b4b396fa4e06349d35c4c46/client/main.scss#L18-20

Comment (ironsidevsquincy) [inline bower.json]: oh yeah, that's right. icky

**Comment by matthew-andrews** [comment]
!!!

**Comment by matthew-andrews** [comment]
you didn't fix it

**Comment by matthew-andrews** [comment]
https://github.com/Financial-Times/next-article/pull/577

**Pull request merged**